GVFS/GVFS.Common/Prefetch/BlobPrefetcher.cs

```diff
@@ -229,38 +229,83 @@ public virtual void Prefetch(string branchOrCommit, bool isBranch)
                 }
             }
 
-            DiffHelper blobEnumerator = new DiffHelper(this.Tracer, this.Enlistment, this.FileList, this.FolderList, includeSymLinks: false);

```

Comment:
Looks like line 214 had an earlier attempt to document the pipeline, but it is very sparse and also now in the wrong place. Remove it.

Reply:
Will be handled in my next PR